app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Reaction Commerce",
+  "description": "Reaction is a modern reactive, real-time event driven ecommerce platform.",
+  "repository": "https://github.com/reactioncommerce/reaction",
+  "logo": "https://avatars1.githubusercontent.com/u/5605462?s=200&v=4",
+  "website": "https://www.reactioncommerce.com",
+  "keywords": ["reaction", "e-commerce", "meteor"],
+  "buildpacks": [
+    {
+      "url": "https://github.com/reactioncommerce/reaction-buildpack.git"
+    }
+  ],
+  "addons": [
+    "papertrail",
```

Comment:
Unless there's a specific reason to choose Papertrail, I'd recommend using Logentries as the default logging system. It leads the pack on alerting capabilities and can alert on patterns, anomalies, and absence of expected patterns.

Reply:
I think it was probably because I chose Papertrail because that's the logging service that I am used to, but I have no problems switching over if it is also free. Papertrail also does all the things you mentioned but I don't have much loyalty to one or the other. Whatever gets the job done